ArchiSteamFarm/Core/Utilities.cs

```diff
@@ -334,11 +334,11 @@ public static class Utilities {
 				return true;
 			}
 
-			if (ForbiddenPasswordWords.Any(word => password.Contains(word, StringComparison.InvariantCultureIgnoreCase))) {
+			if (ForbiddenPasswordPhrases.Any(word => password.Contains(word, StringComparison.InvariantCultureIgnoreCase))) {
 				return true;
 			}
 
-			if (additionallyForbiddenWords.Length > 0 && additionallyForbiddenWords.Any(word => password.Contains(word, StringComparison.InvariantCultureIgnoreCase))) {
+			if ((additionallyForbiddenWords?.Count ?? 0) > 0 && additionallyForbiddenWords!.Any(word => password.Contains(word, StringComparison.InvariantCultureIgnoreCase))) {
```

Comment:
I dislike this style a lot, just do `if (nullable?.Any(something => something) == true)`.